server/liveblog/embed/embed.py

```diff
@@ -11,12 +11,28 @@
 """Embed module"""
 import superdesk
 from flask import render_template, request
+from superdesk import get_resource_service
 
-bp = superdesk.Blueprint('embed_liveblog', __name__, template_folder='templates', static_folder='assets')
+ASSETS_DIR = 'embed_assets'
+bp = superdesk.Blueprint('embed_liveblog', __name__, template_folder='templates', static_folder=ASSETS_DIR)
 
 
 @bp.route('/embed/<blog_id>')
 def embed(blog_id):
-    return render_template('embed.html', blog_id=blog_id, api_host=request.url_root)
+
+    def complete_url(url):
+        def is_relative(url):
+            return not (url.startswith('/') or url.startswith('http://') or url.startswith('https://'))
```

Comment:
"Absolute URLs are URLs that start with a scheme"

/script/main.js is relative.

http://www.coffeecup.com/help/articles/absolute-vs-relative-pathslinks/
http://en.wikipedia.org/wiki/HTTP_location